test/schema.sql

```diff
@@ -193,6 +193,20 @@ CREATE TABLE leaderboard (
   INDEX last_vote (last_vote)
 );
 
+CREATE TABLE skins (
+  id VARCHAR(100) NOT NULL,
```

Comment:
If this is space id, we can remove `skin` in `spaces` right?

Reply:
It's being left there for backward compatibility